### PR TITLE
noobaa: don't set NodeAffinity for standalone

### PR DESCRIPTION
**Pull request by jarrpa**
Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

File: controllers/storagecluster/noobaa_system_reconciler.go [hunk 1]
```diff
@@ -119,9 +119,13 @@ func (r *StorageClusterReconciler) setNooBaaDesiredState(nb *nbv1.NooBaa, sc *oc
 
 	nb.Spec.CoreResources = &coreResources
 	nb.Spec.DBResources = &dbResources
-	placement := getPlacement(sc, "noobaa-core")
-	nb.Spec.Tolerations = placement.Tolerations
-	nb.Spec.Affinity = &corev1.Affinity{NodeAffinity: placement.NodeAffinity}
+	isStandalone := sc.Spec.MultiCloudGateway != nil &&
```
Comment (nbalacha) [inline controllers/storagecluster/noobaa_system_reconciler.go:122]: r.IsNoobaaStandalone already has this information (line 114 in this file)

**Review by nbalacha — CHANGES_REQUESTED** [review]

File: controllers/storagecluster/noobaa_system_reconciler.go (hunk 1 above)
Comment (jarrpa) [reply controllers/storagecluster/noobaa_system_reconciler.go:122]: ...oops! you're right!

**Review by nbalacha — APPROVED** [review]

**Comment by jarrpa** [comment]
Verified that this works on a personal OCP cluster in AWS.

**Pull request merged**